src/index.js

```diff
@@ -0,0 +1,137 @@
+// Temperature & Landscape UI Module
+let isFahrenheit = true;
+
+const convertToCelsius = (f) => Math.round((f - 32) * 5 / 9);
+
+const WeatherUI = {
+  currentTemp: 70, // always stored in Fahrenheit
+
+  updateTemperatureUI() {
+    const tempDisplay = document.getElementById("tempValue");
+    const landscapeImage = document.getElementById("landscapeImage");
+    const caption = document.getElementById("landscapeCaption");
+
+    // Determine what to show
+    const displayTemp = isFahrenheit ? this.currentTemp : convertToCelsius(this.currentTemp);
+    tempDisplay.textContent = `${displayTemp}°${isFahrenheit ? "F" : "C"}`;
+    tempDisplay.className = "";
+
+    // Choose range thresholds based on current unit
+    let tempForRange = isFahrenheit ? this.currentTemp : convertToCelsius(this.currentTemp);
+
+    if ((isFahrenheit && tempForRange >= 80) || (!isFahrenheit && tempForRange >= 27)) {
+      tempDisplay.classList.add("red");
+      landscapeImage.src = "assets/hot.png";
+      caption.textContent = "City landscape - very hot day.";
+    } else if ((isFahrenheit && tempForRange >= 70) || (!isFahrenheit && tempForRange >= 21)) {
+      tempDisplay.classList.add("orange");
+      landscapeImage.src = "assets/warmer.png";
+      caption.textContent = "City landscape - sunny summer day.";
+    } else if ((isFahrenheit && tempForRange >= 60) || (!isFahrenheit && tempForRange >= 15)) {
+      tempDisplay.classList.add("yellow");
+      landscapeImage.src = "assets/warm.png";
+      caption.textContent = "City landscape - blooming spring.";
+    } else if ((isFahrenheit && tempForRange >= 30) || (!isFahrenheit && tempForRange >= 0)) {
+      tempDisplay.classList.add("teal");
+      landscapeImage.src = "assets/cool.png";
+      caption.textContent = "City landscape - rainy cool day.";
+    } else {
+      tempDisplay.classList.add("blue");
+      landscapeImage.src = "assets/cold.png";
+      caption.textContent = "City landscape - cold snowy day.";
+    }
+  },
+
+  increaseTemp() {
+    this.currentTemp += 1;
+    this.updateTemperatureUI();
+  },
+
+  decreaseTemp() {
+    this.currentTemp -= 1;
+    this.updateTemperatureUI();
+  }
+};
+
+// Update the city name in the header live when user types
+document.addEventListener("DOMContentLoaded", () => {
+    const cityNameInput = document.getElementById("cityNameInput");
+    const headerCityName = document.getElementById("headerCityName");
```

Comment:
The indentation is not consistent across this file, sometimes indentation is 2 spaces, sometimes 4. This can be confusing to readers and cause a lot of unnecessary back and forth changes when different members of a team are trying to work on the code. We should choose either 2 spaces or 4 spaces, and use it consistently across the entire project.